frontend/src/AudioAnnotator/AudioAnnotator.js

```diff
@@ -188,6 +195,34 @@ class AudioAnnotator extends Component<AudioAnnotatorProps, AudioAnnotatorState>
             error: undefined,
             annotations,
           });
+
+          /**Presence/Absence mode */
+          if (!!this.state.task && this.state.task.annotationScope === SCOPE_WHOLE) {
+            let useTags = [];
+            let doYouSearchFirstAnnotationTypeTag = true;
+            annotations.map((annotation) => {
```

Comment:
As stated in the [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map), `.map` function creates a new array. Here, you do not use the return variable of the function, but insted on line 205 you mutate a member of this array. As stated on the documentation : *Since map builds a new array, calling it without using the returned array is an anti-pattern*.